src/main/java/com/mindee/CLI.java

```diff
@@ -101,10 +90,32 @@ void passportMethod() throws IOException {
     System.out.println(document.toString());
   }
 
-  MindeeClient getMindeeClient(MindeeSettings mindeeSettings) {
-    return new MindeeClient(new MindeeHttpApi(mindeeSettings));
+  @Command(name = "custom", description = "Invokes a builder API")
+  void customMethod(
+    @Parameters(
+      index = "0",
+      scope = ScopeType.INHERIT,
+      paramLabel = "<path>") File file,
+    @Parameters(
```

Comment:
should be a `"-u, --username"` required option, not a positional argument

Reply:
I do not understand, the file path is mandatory at this point

Reply:
Not sure why it's showing lines above... I put the comment on line 99 which is for `accountName`.

Path parameter does not change.